text/0000-plan-flags.md

```diff
@@ -9,206 +9,238 @@
 # Summary
 [summary]: #summary
 
-We would like to add the ability for buildpacks to be able to specify `build` and `launch` requirements for a dependency.
-To do so we should add `build` and `launch` fields to both the Build Plan (TOML) and Buildpack Plan (TOML) files.
+The `metadata` field that is part of require should be removed and replaced with fields that have defined purpose to standardize inter-buildpack communication.
 
 # Motivation
 [motivation]: #motivation
 The overarching goal of this RFC is to 
 
-1) Simplifies buildpack implementation.
+1) Simplifies buildpack implementation and the  Build Plan (TOML) and Buildpack Plan (TOML) formats.
 
-2) Clarify how buildpacks communicate their dependencies to other buildpacks.
+2) Clarify how buildpacks communicate with each other.
+
+3) Leverage the lifecycle to implement steps that would simplify the buildpack authoring process.
 
 State of Cloudfoundry Buildpacks:
 
-- The current method that the Cloudfoundry Buildpacks use to indicate they rely on an earlier buildpack's dependencies is to set a string `build` and/or `launch` field in the Build Plan (TOML) `requires.metadata` section.
-- the throwing this information in the `requires.metadata` obfuscates the flow of `build` and `launch` flags from dependencies to the Layer Metadata.  
+- The current method that the Paketo Buildpacks uses to communicate information to an earlier buildpack is to set an arbitrary string field in the Build Plan (TOML) `requires.metadata` section.
```

Comment:
Probably part of this RFC is 'establishing a pattern where we can offload generic operation to the lifecycle.'